docs/tutorials/Date Fruit Classification using MindsDB and MongoDB.mdx

```diff
@@ -0,0 +1,165 @@
+## Introduction
+In this tutorial, we’ll create and train a machine learning model, or as **MindsDB** calls it, an `AI Table` or a `predictor`. By querying the model, we’ll predict the class of Date Fruit from these *seven types*: Berhi, Iraqi, Sogay, Deglet, Rotana, Safavi, Dokol. We're going to use **MongoDB** as the data source here.
+
+### Pre-requisites
+1. Make sure you have access to a working MindsDB installation, either locally or at **[MindsDB Cloud](https://cloud.mindsdb.com/)**. If you want to learn how to set up your account at MindsDB Cloud, follow **[this guide](https://docs.mindsdb.com/setup/cloud)**. It takes a few minutes only.
+2. We need to download **MongoDB Database & MongoDB Compass** in order to connect with **MindsDB Cloud.** You can download the **[Database here](https://www.mongodb.com/try/download/community) & [Compass here.](https://www.mongodb.com/try/download/compass)** Do follow their MongoDB Database download **[guide here](https://www.mongodb.com/docs/manual/installation/)**
+
+![mongodb-compass](images/date-fruit-classification/data-setup.png)
+
+3. We'll be using **[Date Fruit Datasets](https://www.kaggle.com/datasets/muratkokludataset/date-fruit-datasets)** from **Kaggle**. You need to download a copy of the dataset locally and extract the `.csv` file from it.
+
+Let’s get started.
+
+
+## Data Setup
+If you're going to use **MindsDB Cloud's files database** for connecting to the MongoDB database, then follow the steps as shown below--
+
+### Upload the dataset to MindsDB Cloud
```

Comment:
Since you are connecting your local MongoDB database, you don't need to upload a file to MindsDB.
Please remove this `Upload the dataset to MindsDB Cloud` section.

Reply:
@martyna-mindsdb This is done too!